pyaedt/modeler/cad/Primitives3D.py

```diff
@@ -91,8 +91,10 @@ def create_box(self, position, dimensions_list, name=None, matname=None):
         >>> box_object = hfss.modeler.primivites.create_box(origin, dimensions, name="mybox", matname="copper")
 
         """
-        assert len(position) == 3, "Position Argument must be a valid 3 Element List"
-        assert len(dimensions_list) == 3, "Dimension Argument must be a valid 3 Element List"
+        if len(position) != 3:
```

Comment:
@dcrawforAtAnsys maybe we could improve the test in test_08_Primitives3D > test_02_create_box

Reply:
@gmalinve: Will try to improve test coverage in a subsequent PR.

Reply:
@dcrawforAtAnsys no worries, we created this PR #3512 for the same purpose.